src/books/dto/book-create.dto.ts

```diff
@@ -0,0 +1,33 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { BookCreate } from '../domain/book-create';
+
+export class BookCreateDto {
+  @ApiProperty()
+  readonly name: string;
+
+  @ApiProperty({ default: true })
```

Comment:
Như comment khác, chỉ đặt giá trị mặc định ở một nơi duy nhất thôi

```suggestion
  @ApiProperty()
```